code/modules/antagonists/paradox_clone/powers/gaze.dm

```diff
@@ -0,0 +1,72 @@
+
```

Comment:
```suggestion
```

Reply:
no.

Reply:
can I ask why?

Reply:
same question why do I need to remove this lol

Reply:
lmao fair. I looked again and it's fine imho, I'd just ask that you be consistent since there are some files there that start with no line and some that start with one. If you could get them all the same that'd be good.